src/SDKs/Automation/Management.Automation/Generated/Models/SyncType.cs

```diff
@@ -12,11 +12,11 @@ namespace Microsoft.Azure.Management.Automation.Models
 {
 
     /// <summary>
-    /// Defines values for StartType.
+    /// Defines values for SyncType.
     /// </summary>
-    public static class StartType
```

Comment:
This is a breaking change for SDK users, please update the version number to `3.4.0-preview`

Reply:
Please update the aseembly file version [here ](https://github.com/Azure/azure-sdk-for-net/blob/9d719904e51dfb1efc1893575d7d3352964a8b99/src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs#L11)

Reply:
Fixed.